app.py

```diff
@@ -0,0 +1,34 @@
+import smtplib
+from typing import Optional
+
+from fastapi import FastAPI, HTTPException, Header
+
+from domain.models import MailBody
+from infrastructure.EmailSender import send
+
+app = FastAPI()
+
+
+@app.get("/")
+async def hello():
+    return {
+        "hello": "hello"
```

Comment:
Mimo wszystko usunąłbym to bo kiepsko wygląda takie losowe hello. Do testów ok ale jak będzie miało pójść do deployu to bym wywalił